examples/all-clusters-app/mbed/CMakeLists.txt

```diff
@@ -77,7 +77,8 @@ target_sources(${APP_TARGET} PRIVATE
                ${ENERGY_MANAGEMENT_COMMON}/src/EnergyEvseManager.cpp
                ${ENERGY_MANAGEMENT_COMMON}/src/DeviceEnergyManagementDelegateImpl.cpp
                ${ENERGY_MANAGEMENT_COMMON}/src/DeviceEnergyManagementManager.cpp
-
+               ${ENERGY_MANAGEMENT_COMMON}/src/EVSEManufacturerImpl.cpp
+               ${ENERGY_MANAGEMENT_COMMON}/src/utils.cpp
```

Comment:
Should this be DEMUtils.cpp?
Minor point - but may be worth putting these in alphabetical order.